api_tests/src/test_environment.ts

```diff
@@ -0,0 +1,119 @@
+import { Script } from "vm";
+import { Config } from "@jest/types";
+import { execSync } from "child_process";
+import commander from "commander";
+import { LedgerRunner } from "../lib/ledgers/ledger_runner";
+import { HarnessGlobal } from "../lib/utils";
+import NodeEnvironment from "jest-environment-node";
+import rimraf from "rimraf";
+import * as fs from "fs";
+import { Mutex } from "async-mutex";
+
+// ************************ //
+// Setting global variables //
+// ************************ //
+
+interface ConfigInterface {
+    ledgers: string[];
+    logDir: string;
+}
+
+export class E2ETestEnvironment extends NodeEnvironment {
```

Comment:
```suggestion
export default class E2ETestEnvironment extends NodeEnvironment {
```
and remove `module.exports = E2ETestEnvironment;` at the end of the file. I believe my proposal is more TypeScript idiomatic.

Reply:
Now my ide complains: 

![image](https://user-images.githubusercontent.com/224613/75826762-59536200-5dfc-11ea-80a7-af57b25a37ff.png)

Reply:
That is normal @bonomat because you are not importing it anywhere, Jest is importing it at runtime :)

Reply:
Yes just ignore your IDE here :)